src/main/flight/mixer.c

```diff
@@ -511,6 +513,13 @@ static float calcEzLandLimit(float maxDeflection)
         ezLandLimit = maxDeflection / mixerRuntime.ezLandingThreshold; // normalised 0 - 1
         ezLandLimit = fmaxf(ezLandLimit, mixerRuntime.ezLandingLimit); // stay above the minimum
     }
+    // calculate limit to where the mixer can raise the throttle based on RPY stick deflection
+    // TODO sanity checks like number of sats, dop, accuracy?
+    if (speed < mixerRuntime.ezLandingSpeed && mixerRuntime.ezLandingSpeed > 0.0f) {
+        const float speed_limit = speed / mixerRuntime.ezLandingSpeed;  // normalised 0 - 1
```

Comment:
maybe 
check (mixerRuntime.ezLandingSpeed > 0.0f) first, maybe with &&, so that if set to zero, nothing happens, and that check is done before the comparison and other code?

Reply:
with speed usually > 0, testing speed < mixerRuntime.ezLandingSpeed first is faster. And with speed >= 0 second test will always be true, but it is better to keep it.